about.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Visit Uganda Summit</title>
+    <link rel="stylesheet" href="about.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=lato"
+      rel="stylesheet"
+    />
```

Comment:
- [OPTIONAL] The template provided font families and a color swatch to be used. Kindly try using them.
  - Fonts used originally: Cocogoose, Lato
  - Basic color swatch: #272A31, #EC5242, #D3D3D3